### DISCUSSION TITLE
bleak alternatives of AssignedNumbers and DefaultDelegate

### DISCUSSION THREAD

**Discussion by ERI-Admin** [hbldh/bleak#944] [General]
* bleak version:0.16
* Python version: Python 3.8
* Operating System: Mac OS
* BlueZ version (`bluetoothctl -v`) in case of Linux:

### Description

I am working on translate the bluepy code to bleak.

bluepy uses only three functions as below.
-Scanner
-AssignedNumbers
-DefaultDelegate

I could find Scanner in bleak, but can't find bleak alternative of the laters.
Is there any good tutorial to transit bluepy to bleak? (Or at least above two functions?)

**Comment by dlech** [comment]
There are docs: https://bleak.readthedocs.io/en/latest/

and examples: https://github.com/hbldh/bleak/tree/develop/examples
